### ISSUE TITLE
Problems with MailcapCommandMap

### ISSUE THREAD

**Issue by morvael** [eclipse-ee4j/jaf#33]
javax.activation.MailcapCommandMap references com.sun.activation.registries.LogSupport and com.sun.activation.registries.MailcapFile. These files are only available to Maven users in old com.sun.activation:javax.activation:1.2.0 dependency, but in turn this dependency contains duplicates of classes available in jakarta.activation:jakarta.activation-api:1.2.1 from the main javax.activation package.

Earlier all those classes were in JDK, but in JDK 11 they were removed, and now there are serious problems when trying to use Java Mail API (which includes registration of non-existent mime type handlers like com.sun.xml.internal.ws.encoding.StringDataContentHandler, which is now available as com.sun.xml.ws.encoding.StringDataContentHandler in jaxws-rt).

All in all, Java Mail API is broken on JDK 11.

**Comment by bshannon** [comment]
I can't quite tell what you're trying to do, but note that there are two jar files
with different purposes and they should never be used together.

The jakarta.activation:jakarta.activation-api jar file is intended **only* for compiling
against.  It defines only the standard API and is suitable when the actual API and
implementation are provided by the runtime, as they are in a Java EE or Jakarta EE
application server.

The jakarta.activation:jakarta.activation jar file contains the complete API and
implementation.  If you're assembling your own application or runtime to use on
JDK 11, this is what you want.  JavaMail works just fine with this jar file on JDK 11.
You'll want at least the 1.2.1 versinon of this jar file, which was updated for some
JDK 11 changes.

Note also that there's a conflict between JavaMail and JAX-WS when registering
MIME type handlers.  You'll need the newest version of JAX-WS to avoid the
conflict.

**Comment by morvael** [comment]
My goal was to use same/similar implementations to those we used previously on Oracle JDK 8 when running on Adopt OpenJDK 11 (tests), and WildFly application server (real application). We had serious problems with some web services when different implementations were used underneath. So I tried to pull all those reference implementations, and there were various conflicts, like the one I have described. Since then, they have all been resolved.

All in all this removal of classes from JDK, transition to new jars, and moving classes to new packages created serious mess in the ecosystem.
